sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/ScriptTransformationExec.scala

```diff
@@ -424,7 +426,9 @@ case class HiveScriptIOSchema (
     recordReaderClass.map { klass =>
       val instance = Utils.classForName(klass).newInstance().asInstanceOf[RecordReader]
       val props = new Properties()
-      props.putAll(outputSerdeProps.toMap.asJava)
+      // props.putAll(outputSerdeProps.toMap.asJava)
+      // see https://github.com/apache/kafka/pull/3647
```

Comment:
ditto. I think it is better to remove commented code and refer to https://github.com/scala/bug/issues/10418.